HipChatAdmin/Public/New-HipchatUser.ps1

```diff
@@ -82,19 +82,23 @@ function New-HipchatUser{
 			Write-Output "Exception Message: $($_.Exception.Message)"
 			Exit 1
 		}
-
-	}
-
-	END {
-
 		# Check response status code #
 		if ($Call.StatusCode -eq '201') {
 			Write-Verbose "User Created Successfully!"
-			Write-Output $Call.StatusCode
+			$OutputObject = New-Object -TypeName PSObject
+			$OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'MentionName' -Value $MentionName
+            $OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'FirstName' -Value $FirstName
```

Comment:
Not sure if it's a GitHub display issue or not, but some of the indentation seems off on each of these changes. I'd like to keep white space and indentation consistent.